feat(content): Add warning when you've upset Hai Sympathizers (#8944)
endless-sky/endless-sky: data/hai/hai missions.txt

mission "Hai Sympathizer Warning"
	invisible
	source
		government "Hai"
		not attributes "uninhabited" "retirement"
	to offer
		"reputation: Hai Merchant (Sympathizers)" < 0
		not "warned about hai merchant politics"
	on offer

> 		log `Got noticed at a spaceport bar as a "famous warrior." Was warned to look out for bounty hunters or pirates hoping to make a name for themselves.`

Reviewer: You will want to update this to reflect the Hai.

Author: ```suggestion
		log `Got spoken to by a Hai merchant at a spaceport bar after a firefight. Was warned about the factions among Hai merchants and the danger of getting in trouble if sympathisers to the Unfettered get caught in the cross-fire when fighting the Unfettered themselves. Learned that, as an outsider, "self-defence" is not an adequate argument where the deaths of Hai citizens are involved.`
```

		set "warned about hai merchant politics"
		conversation
			`As you sit in a spaceport bar waiting for the food that you've ordered, a Hai unceremoniously pulls up a chair opposite you and takes a seat. He looks to be a merchant, and one who has seen more than a few things besides. He squints at you before speaking, "You are Captain <last>, are you not?"`
			choice
				`	"That would be me."`
					goto acknowledge
				`	"I think you have the wrong captain."`

> 			`	He slaps a device on the table and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`

Author: This is the specific comma that MS Word is constantly telling me to take out.

> 			`	He slaps a device on the table and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`

Reviewer 2: MS Word is wrong, and your instincts were right. This is two separate clauses with a conjunction, so it needs that comma.

He slaps device, and projection springs.

Perhaps MS Word may is being confused by the compound predicate of "springs up and begins"?

			label acknowledge

> 			`	He heaves a sigh. "You've been fighting the Unfettered you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers; and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`

Reviewer: ```suggestion
			`	He heaves a sigh. "You've been fighting the Unfettered, you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers; and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
```
`You've been fighting the Unfettered you see` can be taken two ways, but the way it was just didn't read right to me.

> 			`	With a wave of his hand he indicates the spaceport around you. "You might have noticed some difficulty landing potentially. You see because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`

Reviewer: ```suggestion
			`	With a wave of his hand he indicates the spaceport around you. "You might have noticed some difficulty landing potentially. You see, because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
```
`You might have noticed some difficulty landing potentially` What does this mean?

Reviewer 3: if you attack them you wont be able to land on Hai worlds?

Author: ```suggestion
			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
```

> 			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, so you might notice that some people are less offended when you shoot down someone they've been arguing with, but everyone has friends on all sides, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you. Despite that, the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first..."`

Reviewer 3: I love this whole thing but it is a bit of a text wall, is it not?
Would be good to have a way to break it down a bit

Author: It's only 4 lines, and to be honest, he's not interested in letting you ask that question, since it should be obvious by the time he finishes.

Reviewer 3: maybe just say you try to say something but can't quite place a word?
looking at FW for instance even looking at a video had a 'continue watching' "choice"

Author: You are being lectured by someone who has come to you because you messed up.
And again, it's at most 4 lines. We didn't break things up in HR unless they were over 5 lines long.

Reviewer 3: Alright then

> 			`	He looks at you very seriously to make sure what he's said has sunk in.`

Author: I'm, not sure what it's like over there, but under no circumstances would it *ever* be "sunken" for me locally.
Is there a reference you can source for this?

> 			`	He looks at you very seriously to make sure what he's said has sunk in.`

Reviewer 2: The original would be preferred.

For some reason, I had it stuck in my head that an archaic form was the "technically correct" one.

https://www.grammarphobia.com/blog/2019/05/sink-sank-sunk.html

“Historically, the past tense of sink has been both sank and sunk (the boat sank; the boat sunk), and the past participle has been both sunk and sunken (the boat had already sunk; the boat had already sunken). In modern English, the past is generally sank and the past participle is sunk, with the form sunken now surviving only as an adjective, as in a sunken garden or sunken cheeks.”

And even the inpage result from Google:

In modern English, the past is generally sank and the past participle is sunk, with the form sunken now surviving only as an adjective

> 			`	He looks at you very seriously to make sure what he's said has sunk in.`

Reviewer 4: In my subjective en_GB/en_AU experience, 'has sunk in' is the correct form.

			choice
				`	"I understand, I've stepped on some toes."`
					goto trouble
					to display
						"reputation: Hai" < 1
				`	"I understand, thank you for warning me."`
					goto end
					to display
						"reputation: Hai" >= 1

> 				`	"Why should I get in trouble for defending myself against Sympathizers."`

Reviewer: ```suggestion
				`	"Why should I get in trouble for defending myself against Sympathizers?"`
```

			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`

> 			`	"When a report gets made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. As you are not a citizen though, the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a Captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`

Reviewer 3: maybe this could be nuanced that you're not a citizen yet?
or do you really mean that they are this kind of society? because it feels like this could lead to abuse with humans born in Hai space or merchants who came there to live?

Author: As a space-based captain, you're not a citizen full-stop.
They are this kind of society, but there are pathways to citizenship; they just require you to sit still and own something that gives you a vested interest in their society.

This criteria never applies to the player.

Reviewer 3: good to know!
maybe that could be hinted somehow

Author: It's kind of alluded to, but there'll be more things that touch on that later too.
For the purpose of this, you're not a citizen right now, so the specifics about citizenship don't really belong here. It'd distract from the core thrust of this.

> 			`	"When a report gets made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. As you are not a citizen though, the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a Captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`

Reviewer: ```suggestion
			`	"When a report documents that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. But you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
```
I know Merchants are a class in Hai society but I think capitalising `Captain` feels strange.

Author: ```suggestion
			`	"When a report is made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. However, you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
```

			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
			
			label trouble
			branch "less trouble"
				"reputation: Hai" >= 1
				"reputation: Hai (Wormhole Access)" >= 1
				"reputation: Hai Merchant" >= 1
				"reputation: Hai Merchant (Human)" >= 1
			# At least one other group besides the Sympathizers are upset.

> 			`	"You have gotten yourself in trouble Captain."`

Reviewer: If the above isn't met, the text that is displayed would be:
```
"You have gotten yourself in trouble Captain."
He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."
```
Is that right?

Author: yes

> 			`	"You have gotten yourself in trouble Captain."`

Reviewer: I pointed it out because it might be strange to read:
`"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`

`"You have gotten yourself in trouble Captain."`

`He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`

I would remove this line.
```suggestion
```

Reviewer 2: Hmm... We can't just swap the order of the first two paragraphs, because of the other substitutions and things, right?

```"You have gotten yourself in trouble Captain."

"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."

He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."```

Because if we could, that reads even better to me than removing the line.

Author: Oh I see what I've overlooked. Hangon, I'll make it a new suggestion.

			action
				"reputation: Hai" >?= 1
				"reputation: Hai (Wormhole Access)" = "reputation: Hai"
				"reputation: Hai Merchant" >?= 1
				"reputation: Hai Merchant (Human)" >?= 1
			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`

> 			

Reviewer: ```suggestion
				goto end
```

> 				`	"Why should I get in trouble for defending myself against Sympathizers?"`
> 			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
> 			`	"When a report is made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. However, you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
> 			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
> 			
> 			label trouble
> 			branch "less trouble"
> 				"reputation: Hai" >= 1
> 				"reputation: Hai (Wormhole Access)" >= 1
> 				"reputation: Hai Merchant" >= 1
> 				"reputation: Hai Merchant (Human)" >= 1
> 			# At least one other group besides the Sympathizers are upset.
> 			`	"You have gotten yourself in trouble Captain."`
> 			action
> 				"reputation: Hai" >?= 1
> 				"reputation: Hai (Wormhole Access)" = "reputation: Hai"
> 				"reputation: Hai Merchant" >?= 1
> 				"reputation: Hai Merchant (Human)" >?= 1
> 			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`

Author: @EjoThims @bene-dictator that line was meant to come off of the above section directly, and I overlooked that you could run into it from the explanation.
```suggestion
				`	"How could I be in trouble for defending myself against Sympathizers?"`
					to display
						"reputation: Hai" < 1
				`	"Why should I get in trouble for defending myself against Sympathizers?"`
					to display
						"reputation: Hai" >= 1
			action
				set "Hai Sympathizer Warning: Explained Trouble"
			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
			`	"When a report is made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. However, you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
			
			label trouble
			branch "less trouble"
				"reputation: Hai" >= 1
				"reputation: Hai (Wormhole Access)" >= 1
				"reputation: Hai Merchant" >= 1
				"reputation: Hai Merchant (Human)" >= 1
			# At least one other group besides the Sympathizers are upset.
			branch "explained trouble"
				has "Hai Sympathizer Warning: Explained Trouble"
			`	"You have gotten yourself in trouble Captain."`
			label "explained trouble"
			action
				"reputation: Hai" >?= 1
				"reputation: Hai (Wormhole Access)" = "reputation: Hai"
				"reputation: Hai Merchant" >?= 1
				"reputation: Hai Merchant (Human)" >?= 1
			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`
```

			label "less trouble"
			# Only the Sympathizers are upset.
			`	"So far, you have been lucky. Hai Security has taken your history into account, and decided your recent activities are within the tolerable range of responses to the hostile situations you have got yourself into..." He slides a data stick across to you. "Nevertheless, this is a copy of the file on you that states you have now been advised of the situation. If you now proceed to upset Hai Security, they will not provide you with a warning."`

> 			

Reviewer 3: I think you forgot to repair sympathizer reputation here?
or do you never repair that rep intentionaly?

Author: I'm on the fence about it, leaning towards intentionally not repairing it, since you'll only be negative by a very small amount for this option to appear.

Reviewer 3: you could just give +50 relations with them or smth

Author: If it's small enough to be fixed by +50 relations, that's the scenario when I *don't* think anything should be done...

Reviewer 3: What about cutting the rep in two? That way not much changes if its low but otherwise it does more
We could always check for it being lower than smth too

Author: If it's really low and Hai rep isn't - that's a different mission I want to make.

			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, and some people are less offended when you shoot down someone they've been arguing with. Everyone has friends on all sides though, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you, but the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first."`
			`	He looks at you very seriously to make sure what he's said has sunk in.`
			branch "low reputation"
				"reputation: Hai" < 1
			choice
				`	"I understand, thank you for warning me."`
					goto end
				`	"Why should I get in trouble for defending myself against Sympathizers?"`
					goto defending
			label "low reputation"
			choice
				`	"I understand that I've stepped on some toes."`
					goto trouble
				`	"How could I be in trouble for defending myself against Sympathizers?"`
			label defending
			action

> 				set "Hai Sympathizer Warning: Explained Trouble"

Reviewer 5: This should probably be lower case.
Mission autoconditions (or conditions meant to look like them) are capitalised, but others almost always aren't.
I know you're using the mission name here, but it's not a mission autocondition.

Author: While I'm aware that's generally true, a ways back when `to display` was implemented people started using them capitalised where they referred to the mission name and were used only in the mission (vs conditions set for later missions).
e.g.
```
			choice
				`	"The Hai are no longer a secret in human space, but they are concerned how you ended up here."`
					goto revealed
					to display
						not "Hai Reveal [C02] Cleanup: label revealed"
				`	"What happened to the ship that brought you here?"`
					goto kidnappers
					to display
						not "Hai Reveal [C02] Cleanup: label kidnappers"
				`	"Well, let's get you home before anything else happens."`
					goto depart
			label revealed
			action
				set "Hai Reveal [C02] Cleanup: label revealed"`
```
I'm mostly familiar with it having been applied in HR, because that's where I see it.
Having a look around it appears that:
- HR uses capitals.
- Remnant stuff uses lowercase.
- PR'd Korath stuff uses capitals.
- Unfettered PR is currently a mess mixing both capitals and lowercase sometimes within the same condition.

I do kind of like them being capitalised, because it means they show up next to the mission in a save file which makes troubleshooting things easier, but I'm not pressed either way.
Seems like we've got a conflict of convention at present though.

Author: Note: There were at least half a dozen missions rejigged for HR using this before I even learned how to use the syntax for this - this usage didn't originate from me.

Reviewer 3: I thought I was using lower case but I must have slipped then
I used "unfettered: smth" as convention instead of basing them on missions most of the time

Reviewer 6: I'd prefer all lowercase. Looking to https://github.com/endless-sky/endless-sky/pull/7660 where a large number of to display nodes were added, lowercase condition names are used. The condition names are also abbreviated, not being the full name of the mission that the conditions are being used in (such as "saes" for "Sheragi Archaeology: Emerald Sword").

Author: Using the PR of the person posing the possibility as evidence to back up the decision to do so isn't really the most compelling argument.
Is there a tangible benefit that can be defined for using lowercase/abbreviated `to display` conditions?

I would like to know if there's a positive rationale for it, since I'll likely have to go through and standardise them all for HR if this is the direction we take.

Reviewer 6: Manually set conditions. regardless of location, are generally lowercase. This allows mission conditions to be separated from all other conditions in the save file, since the save file conditions are sorted alphabetically with uppercase first.

Reviewer: These conditions should have been unset at the end of the conversations anyway since they are unlikely to be used later on. Probably we should start doing that now before it gets too out-of-hand in the future.

Author: Ah, that's a thought. Is there a simple way to unset all conditions regardless of if used? I was under the impression that just doing `set 'condition' = 0` will leave it as still being there because it's now got an assigned value.

Reviewer 3: clear "condition" I think

mission "Hai Sympathizer Warning"
	invisible
	source
		government "Hai"
		not attributes "uninhabited" "retirement"
	to offer
		"reputation: Hai Merchant (Sympathizers)" < 0

> 		not "warned about hai merchant politics"

Reviewer 7: Are these conditions used anywhere else?

Reviewer 3: I think they're good to have for later

Author: Not yet, but in theory there are a few different ways you could potentially end up receiving warnings which I'd like to expand on over time.
I'd also like to be able to increment it for subsequent warnings after some stuff.
It's a condition I'd *like* to use, and like to not have to make a patch mission for later on.

	on offer

> 		log `Got spoken to by a Hai merchant at a spaceport bar after a firefight. Was warned about the factions among Hai merchants and the danger of getting in trouble if sympathisers to the Unfettered get caught in the cross-fire when fighting the Unfettered themselves. Learned that, as an outsider, "self-defence" is not an adequate argument where the deaths of Hai citizens are involved.`

Reviewer 8: Since Endless Sky is written in American English, the word here should be self-defense.
```suggestion
		log `Got spoken to by a Hai merchant at a spaceport bar after a firefight. Was warned about the factions among Hai merchants and the danger of getting in trouble if sympathisers to the Unfettered get caught in the cross-fire when fighting the Unfettered themselves. Learned that, as an outsider, "self-defense" is not an adequate argument where the deaths of Hai citizens are involved.`
```

> 		log `Got spoken to by a Hai merchant at a spaceport bar after a firefight. Was warned about the factions among Hai merchants and the danger of getting in trouble if sympathisers to the Unfettered get caught in the cross-fire when fighting the Unfettered themselves. Learned that, as an outsider, "self-defense" is not an adequate argument where the deaths of Hai citizens are involved.`

Reviewer 7: ```suggestion
		log `Got spoken to by a Hai merchant at a spaceport bar after a firefight. Was warned about the factions among Hai merchants and the danger of getting in trouble if sympathizers to the Unfettered get caught in the cross-fire when fighting the Unfettered themselves. Learned that, as an outsider, "self-defense" is not an adequate argument where the deaths of Hai citizens are involved.`
```

		set "warned about hai merchant politics"
		conversation
			`As you sit in a spaceport bar waiting for the food that you've ordered, a Hai unceremoniously pulls up a chair opposite you and takes a seat. He looks to be a merchant, and one who has seen more than a few things besides. He squints at you before speaking, "You are Captain <last>, are you not?"`
			choice
				`	"That would be me."`
					goto acknowledge
				`	"I think you have the wrong captain."`
			`	He slaps a device on the table, and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`
			label acknowledge

> 			`	He heaves a sigh. "You've been fighting the Unfettered, you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers; and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`

Reviewer 8: A comma is probably a better choice than a semicolon here, since it's followed by the word "and."
```suggestion
			`	He heaves a sigh. "You've been fighting the Unfettered, you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers, and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
```

> 			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`

Reviewer 8: The player doesn't have optical powers because of the easily annoyed Hai — or at least, I don't think they do...?
```suggestion
			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see, because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
```

> 			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, so you might notice that some people are less offended when you shoot down someone they've been arguing with, but everyone has friends on all sides, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you. Despite that, the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first..."`

Reviewer 7: ```suggestion
			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, and some people are less offended when you shoot down someone they've been arguing with. Everyone has friends on all sides though, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you, but the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first."`
```

			`	He looks at you very seriously to make sure what he's said has sunk in.`
			choice

> 				`	"I understand, I've stepped on some toes."`

Reviewer 7: Unsure, but I don't think the current wording is grammatically correct.
```suggestion
				`	"I understand that I've stepped on some toes."`
```

Author: Debatable, but I don't object to this.

> 			choice
> 				`	"I understand that I've stepped on some toes."`
> 					goto trouble
> 					to display
> 						"reputation: Hai" < 1
> 				`	"I understand, thank you for warning me."`
> 					goto end
> 					to display
> 						"reputation: Hai" >= 1
> 				`	"How could I be in trouble for defending myself against Sympathizers?"`
> 					to display
> 						"reputation: Hai" < 1
> 				`	"Why should I get in trouble for defending myself against Sympathizers?"`
> 					to display
> 						"reputation: Hai" >= 1

Reviewer 5: I don't think this choice block is so complex when not using `to display` to merit its use.
```suggestion
			branch "low reputation"
				"reputation: Hai" < 1
			choice
				`	"I understand, thank you for warning me."`
					goto end
				`	"Why should I get in trouble for defending myself against Sympathizers?"`
					goto defending
			label "low reputation"
			choice
				`	"I understand that I've stepped on some toes."`
					goto trouble
				`	"How could I be in trouble for defending myself against Sympathizers?"`
			label defending
```

Author: Ah, good stuff. This kind of grew organically as I wrote through things, but this is much simpler yeah

			action
				set "Hai Sympathizer Warning: Explained Trouble"

> 			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`

Reviewer 7: ```suggestion
			`	He shrugs. "You shouldn't try to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
```

Author: Based on the wiki of Hai speech:
`They make occasional grammatical errors, particularly leaving out articles or inserting the where they are not needed. (This is to hint that they speak a language like Latin that does not use articles.)`
I generally try and place the negative after the action when random Hai are speaking.

"You shouldn't try to" and "You should try not to" are functionally interchangeable in English, but they do have subtly distinct emphasis, and that emphasis is pronounced in Latin.

In this case he's not just saying "don't try and apply this to us", he's telling you to actively put effort into trying *not* to apply those expectations.
It's not "don't think about it", it's saying "deliberately think about the opposite".

			`	"When a report is made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. However, you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
			
			label trouble
			branch "less trouble"
				"reputation: Hai" >= 1
				"reputation: Hai (Wormhole Access)" >= 1
				"reputation: Hai Merchant" >= 1
				"reputation: Hai Merchant (Human)" >= 1
			# At least one other group besides the Sympathizers are upset.
			branch "explained trouble"
				has "Hai Sympathizer Warning: Explained Trouble"

> 			`	"You have gotten yourself in trouble Captain."`

Reviewer: ```suggestion
			`	"You have gotten yourself in trouble, captain."`
```
or
```suggestion
			`	"You have gotten yourself in trouble, Captain <last>."`
```

			label "explained trouble"
			action
				"reputation: Hai" >?= 1
				"reputation: Hai (Wormhole Access)" = "reputation: Hai"
				"reputation: Hai Merchant" >?= 1
				"reputation: Hai Merchant (Human)" >?= 1

> 			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`

Reviewer 8: In this case, a semicolon's probably better.
```suggestion
			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records; Hai Security certainly will. There will not be future warnings."`
```

> 			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records; Hai Security certainly will. There will not be future warnings."`

Reviewer 7: Why is this merchant willing to stick their head out this far for some rando?

Author: Call him a Good Samaritan, every society has a few.

				goto end
			label "less trouble"
			# Only the Sympathizers are upset.
			`	"So far, you have been lucky. Hai Security has taken your history into account, and decided your recent activities are within the tolerable range of responses to the hostile situations you have got yourself into..." He slides a data stick across to you. "Nevertheless, this is a copy of the file on you that states you have now been advised of the situation. If you now proceed to upset Hai Security, they will not provide you with a warning."`
			
			label end

> 			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone..."`

Reviewer 5: ```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain. If you get into trouble, I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone..."`
```
?

> 			`	The waitress arrives with your food, and the merchant stands up. "Good luck Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`

Reviewer: ```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain <last>. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`
```
or
```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`
```
I prefer the first one because this Hai already knows your name so he might as well say it.

Author: He's telling you off, and telling you that you just got away with some sketchy business because he's vouched for you being an ignorant outsider this one time.
It's not necessarily politic to keep saying your name where less forgiving folk might overhear.

So no, he might as well not say it.

But he's still referring to you using 'captain' as a title, which still makes it "Captain" with capitalisation.
For both of these you could add the comma on it's own, but I can't accept any of these suggestions as they currently stand.

Reviewer 4: Agreed on capitalised Captain as a title.

> 			`	The waitress arrives with your food, and the merchant stands up. "Good luck Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`

Reviewer: ```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`
```

> 			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`

Reviewer 7: ```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone..."`
```
